cli/pub_command.go

```diff
@@ -229,6 +232,58 @@ func (c *pubCmd) doReq(nc *nats.Conn, progress *uiprogress.Bar) error {
 	return nil
 }
 
+func (c *pubCmd) doJetstream(nc *nats.Conn, progress *uiprogress.Bar) error {
+	js, err := nc.JetStream()
+	if err != nil {
+		return err
+	}
+
+	for i := 1; i <= c.cnt; i++ {
+		start := time.Now()
+		body, err := pubReplyBodyTemplate(c.body, "", i)
+		if err != nil {
+			log.Printf("Could not parse body template: %s", err)
+		}
+
+		msg, err := c.prepareMsg(body, i)
+		if err != nil {
+			return err
+		}
+
+		msg.Subject = c.subject
+
+		ack, err := js.PublishMsg(msg)
```

Comment:
`ack, errr:=nc.Request(msg, opts().Timeout)` then parse the ack with `jsm.ParsePubAck` and trace the ack